samples/default-authorizationserver/src/test/java/sample/DefaultAuthorizationServerApplicationTests.java

```diff
@@ -67,6 +74,16 @@ public void setUp() {
 		this.webClient.getCookieManager().clearCookies();	// log out
 	}
 
+	@Test
+	void authorizationRequestProperlyEncoded() {
```

Comment:
Please remove this test since it directly tests `UriComponentsBuilder`, which is not owned by this project.